WordPress/src/main/java/org/wordpress/android/ui/accounts/login/Login2FaFragment.java

```diff
@@ -217,6 +217,9 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
             mOldSitesIDs = savedInstanceState.getIntegerArrayList(KEY_OLD_SITES_IDS);
         } else {
             AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_TWO_FACTOR_FORM_VIEWED);
+            if (mService != null) {
+                AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_SOCIAL_2FA_NEEDED);
+            }
```

Comment:
Is the `LOGIN_SOCIAL_2FA_NEEDED` stat meant to be tracked when the two-factor authentication screen is shown during a social login?  If so, the [`LoginActivity.needs2faSocial`](https://github.com/wordpress-mobile/WordPress-Android/blob/c32b10a88b8a2aa31c9f1f0cd60d25b4a4d4e45d/WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java#L323-L329) and [`LoginActivity.needs2faSocialConnect`](https://github.com/wordpress-mobile/WordPress-Android/blob/c32b10a88b8a2aa31c9f1f0cd60d25b4a4d4e45d/WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java#L331-L335) methods would be better locations for that.  Those methods are called when the `Login2FaFragment` is shown from `LoginGoogleFragment` or `LoginEmailPasswordFragment` (i.e. the only places two-factor authentication can be shown from social login).

Reply:
Ah! Def a much better location. Will do!